server/clip_server/model/fast_tokenizer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import gzip
+from functools import lru_cache
+from rust_tokenizers import PyOpenAiGptTokenizer
+from clip_server.helper import __resources_path__
+
+
+@lru_cache()
+def default_vocab():
+    vocab_path = os.path.join(__resources_path__, 'vocab.json')
+    if not os.path.isfile(vocab_path):
+        import json
+        from simple_tokenizer import bytes_to_unicode
+
+        merges = open(default_merges()).read()
+        merges = merges[1 : 49152 - 256 - 2 + 1]
+        merges = [tuple(merge.split()) for merge in merges]
+        vocab = list(bytes_to_unicode().values())
+        vocab = vocab + [v + '</w>' for v in vocab]
+        for merge in merges:
+            vocab.append(''.join(merge))
+        vocab.extend(['<[startoftext>', '<endoftext>'])
```

Comment:
Could you double check the special tokens here? In `simple_tokenizer`, they are using `'<|startoftext|>', '<|endoftext|>'`